README.md

```diff
@@ -61,92 +62,35 @@ Once you've provided the profile, you can deploy the CloudFormation stack using
+See our [CONTRIBUTING.md](CONTRIBUTING.md) guide.
 
-There are two main parts of this project:
 
-1. [quickstart-mongodbatlas](quickstart-mongodbatlas) This is a mirror of https://github.com/aws-quickstart/quickstart-mongodb-atlas. 
-
-2. [cfn-resources](cfn-resources) A set of AWS CloudFormation custom resource providers for MongoDB Atlas Resources. Currently, AWS requires users to manually deploy these resources in each AWS region one one desires to use them in. We support this workflow through the standard AWS cfn submit tooling. Scripts and Github actions are contained in this repository which demonstrate automating this deployment process.
-
-
-## Registering resources to run locally 
-
-1. Please check that you satisfy all the [requirements](#Requirements) before proceeding.
-2. Clone this repo. 
-3. A helper script `[cfn-resources/cfn-submit-helper.sh](cfn-submit-helper.sh)` will build and submit each resource for you. You can also run the `cfn submit` tool yourself. Note- this step will soon not be required when AWS launch a public registry. 
-
-The following command builds and registers all the types used in the quickstart:
-
-```
-cd mongodbatlas-cloudformation-resources\cfn-resources
-./cfn-submit-helper.sh project cluster database-user project-ip-access-list network-peering
-```
-
-## IAM Access Error When Previsioning Resources 
-If you are having difficulty with IAM access, suggest try registering first with the following IAM role [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/execute-role.template.yml). This activates the public registry extensions by first using the private registry extensions.
-
-The naming scheme  for a MongoDB Atlas resource on the AWS CloudFormation Third-Party Public Registry is "MongoDB::Atlas::[RESOURCE-NAME]". 
-
-# Common Troubleshooting when using AWS CloudFormation/CDK with MongoDB Atlas Resources  
-1. Activate the 3rd party extension for each resource (i.e. MONGODB::ATLAS::[RESOURCE-NAME]) in each AWS region and from each AWS account that you wish to deploy from
-2. Ensure you have sufficiently strong AWS IAM Activation Role attached to each 3rd party extension. For sample IAM Role see [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/execute-role.template.yml)
-3. Ensure your activated 3rd party public extension matches name exactly to MONGODB::ATLAS::[RESOURCE-NAME] (you may need to delete private extension if this namespace is already occupied)
+## Troubleshooting
+The following are common issues encountered when using AWS CloudFormation/CDK with MongoDB Atlas Resources: 
+1. Activate the 3rd party extension for each resource (i.e. MONGODB::ATLAS::[RESOURCE-NAME]) in each AWS region and from each AWS account that you wish to deploy.
+2. Ensure you have sufficiently strong AWS IAM Activation Role attached to each 3rd party extension. 
```

Comment:
can we provide additional guidance here? `Sufficiently strong` leaves a lot of room for interpretation